src/dropdown.test.aria.ts

```diff
@@ -205,34 +302,3 @@ test('tooltip', async ({ page }, test) => {
     name: `${test.titlePath.join('.')}.yml`,
   });
 });
-
-test(`value="['one']"`, async ({ page }, test) => {
```

Comment:
Moved setting `value` to `test.describe('multiple=${true}')` and `test.describe('multiple=${false}')`  because we already have to test `multiple` in various states.